### PR TITLE
Frontend storefront fix himel 5

### PR DESCRIPTION
**Pull request by SeefatHimel**
# Description

(Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.)

## Type of change

BC-484 - https://pm23.atlassian.net/browse/BC-484
BC-435 - https://pm23.atlassian.net/browse/BC-435
BC-473 - https://pm23.atlassian.net/browse/BC-473
BC-505 - https://pm23.atlassian.net/browse/BC-505
BC-506 - https://pm23.atlassian.net/browse/BC-506

(Please delete options that are not relevant.)

- [x] Link to a **GH Issue** or **Jira** task here for reference.
- [x] Bug fix
- [x] New feature

# How Has This Been Tested?

(Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration)

- [x] Tested by myself
- [ ] Unit test

# Developer Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# Review check list:

- [ ] Separation of Concerns followed (Reviewer name/NaN)
- [ ] Code is in sync with existing code patterns/technologies. (Reviewer name/NaN)
- [ ] Are there any obvious optimizations that will improve performance? (Reviewer name/NaN)
- [ ] Can any logging or debugging code be removed? (Reviewer name/NaN)
- [ ] Are all data inputs checked (for the correct type, length, format, and range) and encoded? (Reviewer name/NaN)
- [ ] Are invalid parameter values handled? (Reviewer name/NaN)
- [ ] Is error handling done the correct way? (Reviewer name/NaN)
- [ ] Are all the edge cases (null, 0, negative) handled properly? (Reviewer name/NaN)

### CODE REVIEW

File: packages/storefront/components/auth/withAuth.tsx [hunk 1]
```diff
@@ -15,9 +18,22 @@ const WithAuth = (Component: React.FC) => {
       // );
       window.location.href = '/account/sign-in';
```
Comment (asaduzzamanbs23) [inline packages/storefront/components/auth/withAuth.tsx]: @SeefatHimel

File: packages/storefront/components/auth/withAuth.tsx [hunk 2]
```diff
@@ -15,9 +18,22 @@ const WithAuth = (Component: React.FC) => {
...
     }
+    const pushRouter = () => {
+      router.push('/account/sign-in');
+    };
+    useEffect(() => {
+      if (!isLoggedIn) {
+        // return (
+        //   <SignIn />
+        // );
+        window.location.href = '/account/sign-in';
```
Comment (asaduzzamanbs23) [inline packages/storefront/components/auth/withAuth.tsx]: @SeefatHimel

File: packages/storefront/components/checkout/paymentPage/subsections/paymentDetails.tsx [hunk 3]
```diff
@@ -249,78 +250,96 @@ const PaymentDetails: NextComponentType = () => {
...
+                      <label htmlFor="paymentMethodCard" className="pl-2">
+                        Credit card
+                      </label>
+                    </div>
+                    {formikprops.values.paymentMethod === 'Credit card' ? (
+                      <div className="mt-5 rounded border border-gray-300">
+                        {/* <div className="border-b-1 flex flex-wrap items-center justify-between p-4">
+                          <p className="text-sm font-semibold">Credit card</p>
+                          <CreditCard />
+                        </div> */}
```
Comment (asaduzzamanbs23) [inline packages/storefront/components/checkout/paymentPage/subsections/paymentDetails.tsx]: @SeefatHimel remove commented code

File: packages/storefront/components/checkout/paymentPage/subsections/paymentDetails.tsx [hunk 4]
```diff
@@ -335,16 +354,7 @@ const PaymentDetails: NextComponentType = () => {
...
-                          type="radio"
-                          name="shippingAddressPicked"
-                          value="sameShippingAddress"
-                          className="mx-4 mb-4 checked:accent-black"
-                          onClick={() => {
-                            handleSameAddress();
-                          }}
-                          required
-                        />
+                       w
```
Comment (asaduzzamanbs23) [inline packages/storefront/components/checkout/paymentPage/subsections/paymentDetails.tsx:350]: @SeefatHimel is it ok?

File: packages/storefront/components/global/components/cartToast.tsx [hunk 5]
```diff
@@ -9,16 +9,16 @@ interface Props {
 const CartToast: React.FC<Props> = ({ product }: Props) => {
   return (
     <>
-      <div className="flex flex-row ">
-        <div className="col-span-2">
+      <div className="flex flex-row">
+        <div className="">
```
Comment (asaduzzamanbs23) [inline packages/storefront/components/global/components/cartToast.tsx]: @SeefatHimel remove this unused div

File: packages/storefront/components/wishlist/index.tsx [hunk 6]
```diff
@@ -57,19 +57,27 @@ const WishlistComponent: NextComponentType = () => {
...
     }
   };
 
   // async function handleClick(data: string) {
   //   try {
   //     await userAPI.deleteWishlistItem(data);
-  //     toast.success('Item removed from wishlist');
+  //  toast.success('Item removed from wishlist', {
+  //     containerId: 'bottom-right',
+  //   });
```
Comment (asaduzzamanbs23) [inline packages/storefront/components/wishlist/index.tsx]: @SeefatHimel remove commented code

File: packages/storefront/components/wishlist/index.tsx [hunk 7]
```diff
@@ -57,19 +57,27 @@ const WishlistComponent: NextComponentType = () => {
...
   //     dispatch(deleteItemFromWishlist(data));
   //   } catch (error) {
-  //     toast.error('Failed to remove item from wishlist');
+  //  toast.error('Failed to remove item from wishlist', {
+  //     containerId: 'bottom-right',
+  //   });
```
Comment (asaduzzamanbs23) [inline packages/storefront/components/wishlist/index.tsx]: @SeefatHimel remove commented code

**Review by asaduzzamanbs23 — CHANGES_REQUESTED** [review]
@SeefatHimel 
1. Remove all commented code
2. Remove all unused div
3. Remove all unused line

File: packages/storefront/components/checkout/paymentPage/subsections/paymentDetails.tsx (hunk 4 above)
Comment (SeefatHimel) [reply packages/storefront/components/checkout/paymentPage/subsections/paymentDetails.tsx:350]: Yes vai , last time you told us to comment this section (the address section in payment) .

File: packages/storefront/components/wishlist/index.tsx (hunk 7 above)
Comment (SeefatHimel) [reply packages/storefront/components/wishlist/index.tsx]: Done vai

File: packages/storefront/components/wishlist/index.tsx (hunk 6 above)
Comment (SeefatHimel) [reply packages/storefront/components/wishlist/index.tsx]: Done vai

File: packages/storefront/components/global/components/cartToast.tsx (hunk 5 above)
Comment (SeefatHimel) [reply packages/storefront/components/global/components/cartToast.tsx]: Done vai

**Pull request merged**